source/class/qx/tool/cli/api/CompilerApi.js

```diff
@@ -56,7 +56,7 @@ qx.Class.define("qx.tool.cli.api.CompilerApi", {
     
     /**
      * Register compiler tests
-     * @param  cmd: {qx.tool.cli.commands.Command} current command
+     * @param {*} cmd current command
      * @return {Promise<void>}
```

Comment:
So the linter does not accept  hinting at the type of the resolved Promise value?

Reply:
i think that hinting is non standard, it's one of my bad habits and the function should be expressed as `async` and no return value